src/api/wampApi.ts

```diff
@@ -21,6 +21,8 @@ export class WampApi {
 
   private subscriptions: Map<string, Subscription> = new Map();
 
+  private timer: any;
```

Comment:
Type should be a number

Reply:
Deny.

`private timer: number;`
![image](https://user-images.githubusercontent.com/33485080/42081036-1459d47a-7b8d-11e8-89ac-720ac8cb9864.png)

`private timer: NodeJS.Timer;`
![image](https://user-images.githubusercontent.com/33485080/42081120-528a3064-7b8d-11e8-89a6-641cef5bdc7c.png)